### ISSUE TITLE
Watcher while jobs are idle

### ISSUE THREAD

**Issue by antipop001** [jbruce12000/kiln-controller#150]
Hello. I have gotten the watcher script running. I am running it on anohter machine. The curl is good while the a job is running. How do I keep it from sending errors when the kiln jobs are idle?

**Comment by jbruce12000** [comment]
I never intended to run the watcher without an actively running kiln job. To support this would require code changes to the watcher itself so that it does not do any checks until it sees a schedule has started.

**Comment by jbruce12000** [comment]
yes. I usually run it on another machine in my network. it can be run locally on the raspberry pi though.
